### DISCUSSION TITLE
Add Startpage as default search engine option!

### DISCUSSION THREAD

**Discussion by brudermusscode** [zen-browser/desktop#3267] [Ideas]
Have just switched to Zen Browser and didn't see the option for Startpage.com as a default search engine. Would love this to be added!

Thank you friends and **awesome work so far!**

**Comment by rubaboo** [comment]
You mean as one of the options, right? Not the *default* option?

**Reply by brudermusscode** [reply]
As one of the options to choose from when setting it up for the first time!
